README.md

```diff
@@ -124,7 +124,7 @@ ember install ember-metrics
 
 - Ember.js v3.20 or above
```

Comment:
as a follow up, we may bump support for 3.28+ which should simplify CI and should not affect many (if any) users as 3.20 -> 3.28 upgrade should be pretty simple

Reply:
I have done that in https://github.com/adopted-ember-addons/ember-metrics/pull/549.